feat: export `ESLint.defaultConfig` (#18983)
eslint/eslint: docs/src/integrate/nodejs-api.md

This is a static property.

### ◆ ESLint.defaultConfig

```js
const defaultConfig = ESLint.defaultConfig;
```

> The default configuration that ESLint uses internally. This can change between versions, so it's not recommended to rely on this object.

Reviewer: ```suggestion
The default configuration that ESLint uses internally. This can change between versions, so relying on this object is not recommended.
```

Author: Updated 👍🏻
